server/src/main/resources/config/windows/nodeConfig.json

```diff
@@ -29,6 +29,7 @@
         "port": 5555,
         "host": "ip",
         "register": true,
-        "hubPort": 4444
+        "hubPort": 4444,
+        "nodeRecycleThreadWaitTimeout": 300
```

Comment:
Since these values are relevant only to SeLion (and not selenium), I think the SeLionConfig.json file is preferred here.
I'm not seeing the advantage to have this in nodeConfig.